big_tests/tests/domain_removal_SUITE.erl

```diff
@@ -197,6 +202,33 @@ private_removal(Config) ->
         ?assert_equal_extra(<<>>, Val2, #{stanza => Res2})
       end).
 
+bobs_default_groups() -> [<<"friends">>].
+
+bobs_default_name() -> <<"Bobby">>.
+
+roster_removal(Config) ->
```

Comment:
Can you provide tests case where tables `rosterusers`/`rostergroups`/`roster_version` will checking? Eg: check count of `rosterusers` before and after remove domain?

Reply:
I think we could check it in the same test case... just add calls to a function similar to `select_affs_by_room_id`.